android/java/org/chromium/chrome/browser/crypto_wallet/fragments/ApproveTxBottomSheetDialogFragment.java

```diff
@@ -352,8 +353,10 @@ private void approveTransaction() {
         if (txService == null) {
             return;
         }
-        txService.approveTransaction(CoinType.ETH, mTxInfo.id, success -> {
+        txService.approveTransaction(CoinType.ETH, mTxInfo.id, (success, error, errorMessage) -> {
             assert success;
+            Utils.warnWhenError(ApproveTxBottomSheetDialogFragment.TAG_FRAGMENT,
+                    "approveTransaction", error, errorMessage);
```

Comment:
As I understood we need to show that error in UI right? `warnWhenError` just logs it. You can create a follow up to show that on UI on Android